vardict.wdl

```diff
@@ -45,6 +50,12 @@ task VarDict {
         ~{true="var2vcf_paired.pl" false="var2vcf_valid.pl" defined(normalBam)} \
         -N "~{tumorSampleName}~{"|" + normalSampleName}" \
         ~{true="" false="-E" defined(normalBam)} \
+        -M \
+        -A \
```

Comment:
Could you add inputs so these could be unset if desired?

You can look at how the `-E` flag is handled as an example, though these inputs should be Boolean type so `defined` won't be necessary.